apps/omg_status/lib/status.ex

```diff
@@ -68,9 +68,9 @@ defmodule OMG.Status do
   @spec is_enabled?() :: boolean()
   defp is_enabled?() do
```

Comment:
How about `is_enabled?()` returning `true | false | nil` and doing `is_enabled()? || false` when calling?